demo/component/AreaChart.tsx

```diff
@@ -313,27 +315,23 @@ export default class AreaChartDemo extends React.Component<any, any> {
             <YAxis type="category" dataKey="name" />
             <XAxis type="number" xAxisId={0} orientation="top" />
             <XAxis type="number" xAxisId={1} orientation="bottom" />
-            <Area dataKey="uv" type="monotone" stroke="#ff7300" fill="#ff7300" strokeWidth={2} xAxisId={0}
-            />
-            <Area dataKey="pv" type="monotone" stroke="#387908" fill="#387908" strokeWidth={2} xAxisId={1}
-            />
+            <Area dataKey="uv" type="monotone" stroke="#ff7300" fill="#ff7300" strokeWidth={2} xAxisId={0} />
+            <Area dataKey="pv" type="monotone" stroke="#387908" fill="#387908" strokeWidth={2} xAxisId={1} />
             <Tooltip />
           </AreaChart>
         </div>
 
         <p>AreaChart with custom tooltip</p>
         <div className="area-chart-wrapper">
-          <AreaChart width={900} height={250} data={data} margin={{ top: 10, right: 30, bottom: 10, left: 10 }}
-          >
+          <AreaChart width={900} height={250} data={data} margin={{ top: 10, right: 30, bottom: 10, left: 10 }}>
             <XAxis dataKey="name" />
             <YAxis tickCount={7} />
             <Tooltip content={<CustomTooltip external={data} />} />
             <CartesianGrid stroke="#f5f5f5" />
             <ReferenceArea x1="Page A" x2="Page E" />
             <ReferenceLine y={7500} stroke="#387908" />
             <ReferenceDot x="Page C" y={1398} r={10} fill="#387908" isFront />
-            <Area type="monotone" dataKey="pv" stroke="#ff7300" fill="#ff7300" fillOpacity={0.9}
-            />
+            <Area type={stepAround} dataKey="pv" stroke="#ff7300" fill="#ff7300" fillOpacity={0.9} />
```

Comment:
As we already have a story for the AreaChart, could you please add an example with this custom type to storybook?
```
<AreaChart data={data}>
    <Area type={stepAround} dataKey="pv" stroke="#ff7300" fill="#ff7300" fillOpacity={0.9} />
</AreaChart>
```

Reply:
That would also provide a test for this feature, once we have the storybook test runner up.

Reply:
Opened PR
https://github.com/ckifer/recharts/pull/1